tests/kakuro/TestPlayerModel.java

```diff
@@ -8,9 +8,9 @@
 import org.junit.Test;
 import kakuro.PlayerModel;
 
-//iteration 3
-
-public class TestPlayerModel {
+	//iteration 3
```

Comment:
maybe it's good to remove iteration 3 comment cause it's confusing since we did it and it's in second

Reply:
Change it